tensorflow/compiler/xla/service/gpu/gpu_executable.cc

```diff
@@ -46,18 +46,16 @@ using tensorflow::tracing::ScopedAnnotation;
 // Implementation note: HLO profiling is always enabled for GPU executables,
 // since we can use timers around thunks.
 GpuExecutable::GpuExecutable(
-    const string& ptx, const std::vector<uint8>& cubin,
-    std::pair<int, int> compute_capability,
+    const string& text, const std::vector<uint8>& binary,
```

Comment:
"text" is kind of generic.  Could we call it "assembly_text" or "asm_text"?

Reply:
let me revise it as `asm_text`